tests/_support/Helper/TribeDb.php

```diff
@@ -0,0 +1,309 @@
+<?php
```

Comment:
Can't add / call this from common?

Reply:
https://github.com/moderntribe/tribe-common/pull/1522/files